src/getCoinCombination.test.js

```diff
@@ -7,4 +7,25 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`throw an error if 'cents' is negative non-integer number`, () => {
+    expect(() => getCoinCombination(-1.1))
+      .toThrow();
+  });
+
+  it(`return '[1, 0, 0, 0]' if 'coins' = 1`, () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it(`return '[1, 1, 0, 0]' if 'coins' = 6`, () => {
```

Comment:
The test description should be more precise. Consider rephrasing it to something like: 'should return [1, 1, 0, 0] if cents is 6'.